openmetadata-ui/src/main/resources/ui/src/components/Alerts/AlertDetails/AlertDiagnosticInfo/AlertDiagnosticInfoTab.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Card, Col, Input, Row, Tooltip } from 'antd';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { GRAYED_OUT_COLOR } from '../../../../constants/constants';
+import './alert-diagnostic-info-tab.less';
+import { AlertDiagnosticInfoTabProps } from './AlertDiagnosticInfoTab.interface';
+
+interface DiagnosticItem {
+  key: string;
+  value: string | number | boolean;
+  description: string;
+}
+
+const DiagnosticItemRow = ({ item }: { item: DiagnosticItem }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Col span={12}>
+      <Row align="middle">
+        <Col className="d-flex items-center" span={10}>
+          <span className="d-flex items-center gap-1">
+            <p className="text-grey-muted m-0">{t(item.key)}:</p>
+            <Tooltip placement="bottom" title={item.description}>
```

Comment:
Let's not create a separate component if it just renders without any other operations. Move this to the render of the main component itself.